DBTP-289 Add allowed_hosts and database Django utility functions (#1)
uktrade/copilot-python: README.md

# DBT Copilot Django

> A set of utility functions for running Django apps in AWS ECS via AWS Copilot.

Reviewer: Is it worth adding some basic documentation of what the functions are here? Maybe not much more that a simple list?

Author: Yes

# DBT Copilot Django

A set of utility functions for running Django & Flask apps in AWS ECS via AWS Copilot.

## Using  dbt-copilot-python

### Installation

```
pip install dbt-copilot-python
```

### Usage

In settings.py:

1. Add the container IP to ALLOWED_HOSTS so that the load balancer/ALB healthcheck will succeed: 
```

> from dbt_copilot import get_aws_ecs_container_ip
> 
> ALLOWED_HOSTS = [...]
> 
> container_ip = get_ecs_container_ip()
> 
> if container_ip
>     ALLOWED_HOSTS.append(container_ip)

Reviewer: I wonder if it's worth wrapping this up a bit so that in settings we just need...

```suggestion
from copilot-python import add_container_ip_to_allowed_hosts

add_container_ip_to_allowed_hosts(ref ALLOWED_HOSTS)
```

Author: Yes that would work.  

Though it'd be nicer to be a bit more explicit with an assignment, e.g 

```
ALLOWED_HOSTS = [...]
ALLOWED_HOSTS = copilot_setup_allowed_hosts(ALLOWED_HOSTS)
```

Which will end up being used like this:

```
ALLOWED_HOSTS = copilot_setup_allowed_hosts(env.list("ALLOWED_HOSTS"))
```

Reviewer: Makes sense. I was umming and ahhing about assignment vs reference myself. Couldn't make up my mind.
